src/js.rs

```diff
@@ -255,3 +255,17 @@ extern {
     #[wasm_bindgen(method, js_name = propertyIsEnumerable)]
     pub fn property_is_enumerable(this: &Object, property: &JsValue) -> bool;
 }
+
+// String
+#[wasm_bindgen]
```

Comment:
Just a note that while using VS Code it complains with this message: 
```
custom attribute panicked help: message: malformed #[wasm_bindgen] attribute: ParseError(Some("failed to parse all tokens"))
```

Though aside from that it all seems to work.